components/ui/Footer.tsx

```diff
@@ -31,24 +32,22 @@ const FooterNewsletterSub: FC<{ title: string }> = ({ title }) => {
 
   return (
     <form className="flex flex-col w-full" onSubmit={handleSubmit(subscribe)}>
-      <h6 className="text-center md:text-start font-semibold mb-ztg-30">
-        {title}
-      </h6>
-      <div className="flex gap-3 items-center h-ztg-40 mb-auto w-full">
+      <h6 className="font-semibold mb-[26px]">{title}</h6>
+      <div className="flex gap-3 items-center h-[56px] mb-auto w-full">
         <input
           {...register("email", { required: true, pattern: /^\S+@\S+$/i })}
-          className={`h-full grow rounded text-sky-600 p-2 text-ztg-12-120 bg-anti-flash-white border-[1px] focus:outline-none ${
+          className={`h-full grow rounded-[5px] text-sky-600 p-2 text-ztg-12-120 bg-anti-flash-white border-[1px] focus:outline-none ${
             invalid ? "border-vermilion" : "border-none"
           }`}
           type="email"
         />
         <button
           type="submit"
-          className={`h-full w-ztg-40 flex-shrink-0 rounded-full center bg-pastel-blue text-white ${
+          className={`h-full flex-shrink-0 text-ztg-16-150 rounded-full center bg-ztg-blue text-white px-[20px] ${
```

Comment:
> Same as my other comment. Can it be done with standards and not custom values?

In this case I can use `px-5` instead of `px-[20px], but what if we for some reason decide to change font-size to something else than 16px? And IMO `px-[20px]` is more readable than `px-5`. you need to look up tailwind documentation every time design needs to be implemented or changed.
Anyway think this question is better suited for Mikhail. At the beginning of the project I was asking can we make consistent spacings for paddings and similar or at least use default values from tailwind, but that didn't work out, so I just created this inside tailwind config file
```
const spacings = [...new Array(101)]
  .map((_, idx) => idx)
  .reduce((prev, val) => {
    return {
      ...prev,
      [`ztg-${val}`]: `${val}px`,
    };
  }, {});
  ```
  which turned into using `[xx-px]` everywhere.

Reply:
Btw if your not using the tailwind vs code extension you might find it useful, it lets you hover to see what css is applied
<img width="328" alt="image" src="https://user-images.githubusercontent.com/4950844/233376326-2df91c7d-d1d0-418b-a9c2-40bc19a7ac3c.png">